drakrun/drakrun/main.py

```diff
@@ -630,47 +655,23 @@ def process(self, task: Task) -> None:
         if not task.headers.get("execute", True):
             return
 
-        # Gather basic facts
+        timeout = self.timeout_for_task(task)
+        if timeout > MAX_TASK_TIMEOUT:
+            self.log.error(
+                "Tried to run the analysis for more than hard limit of %d seconds",
+                MAX_TASK_TIMEOUT,
+            )
+            return
```

Comment:
I would crash that task with an exception instead of soft fail.

Reply:
It will still generate an alert in our logs, so I'll keep it as it is for now. But I'll think about the error handling here (there are other "soft fails" here that maybe should crash with exception)